### PR TITLE
Checkout 698 - Enable styling of UCO via Stencil CLI

### PR DESCRIPTION
**Pull request by icatalina**
This PR should enable Stencil CLI to render `Optimized Checkout` correctly and inject the `optimized-checkout.css` file when modified.

@mcampa @lord2800

### CODE REVIEW

File: server/plugins/renderer/renderer.module.js [hunk 1]
```diff
@@ -181,6 +181,9 @@ internals.parseResponse = function (bcAppData, request, response, responseArgs,
     var url = responseArgs.url;
 
     if (!_.has(bcAppData, 'pencil_response')) {
+
+        delete response.headers['x-frame-options'];
```
Comment (lord2800) [inline server/plugins/renderer/renderer.module.js]: Hmm. What breaks without this?

**Comment by icatalina** [comment]
@jordanwink201 might be handy to test your stuff...

File: server/plugins/renderer/renderer.module.js (hunk 1 above)
Comment (icatalina) [inline server/plugins/renderer/renderer.module.js]: Injecting UCO within theme editor through `stencil-cli`

Comment (lord2800) [inline server/plugins/renderer/renderer.module.js]: Got it.

File: server/plugins/renderer/responses/raw-response.js [hunk 2]
```diff
@@ -14,17 +14,22 @@ function RawResponse(data, headers, statusCode) {
 
     this.respond = function (request, reply) {
         var response;
-        var paylaod = Buffer.isBuffer(data) ? data.toString('utf8') : '';
+        var payload = Buffer.isBuffer(data) ? data.toString('utf8') : '';
```
Comment (lord2800) [inline server/plugins/renderer/responses/raw-response.js]: 🙇 thanks for the typo fix!

**Comment by lord2800** [comment]
I'm happy with this once you get Travis passing. 👍

**Comment by icatalina** [comment]
Hi @lord2800,

Seems like we're no longer testing for node 0.12.x in Travis, therefore the build passes just fine now with no changes.

I've also fixed a typo on the test `Respomse` -> `Response`.

Could you please confirm you're happy with the tool not working on node 0.12.x?

Thanks! :D

**Comment by lord2800** [comment]
Fairly sure stencil-cli is 4.x+ only now, so I'm happy for this to go through. 👍

**Comment by icatalina** [comment]
You can't load UCO using stencil-cli with the theme editor....

**Pull request merged**